package.json

```diff
@@ -50,13 +52,31 @@
     "docs:watch": "gitbook install && gitbook serve",
     "prepublish": "nwb build",
     "release": "semantic-release pre && npm publish && semantic-release post",
-    "test": "node umd/skatejs.js && semistandard && npm run test:ts && nwb test -- --ci",
-    "test:ts": "tsfmt -r && tsc -p ./",
-    "test:watch": "nwb test --server"
+    "pretest": "npm run style && npm run test:node && npm run test:ts",
```

Comment:
sure it "creates unnecessary cognitive load", but it makes things granular and easy to refactor/maintain.

For instance: a new contributor tries to push code to make PR. suddenly commit will not pass because he made some typescript or js lint errors. Now if the user goes to `package.json` he founds there `style:fix` or just `style:ts:fix` which will fix his issues, instead of need to discover how does the particular project linting et all -> going to particular linter/formatter page and learn how to autofix problems...